redfish-core/lib/oem/ibm/system_attention_indicator.hpp

```diff
@@ -0,0 +1,144 @@
+#pragma once
+
+#include "async_resp.hpp"
+#include "dbus_utility.hpp"
+
+#include <array>
+#include <memory>
+#include <string_view>
+#include <variant>
```

Comment:
`<variant>` is not used.
But `<string>` is used

Reply:
Done